rebar.config

```diff
@@ -77,32 +91,46 @@
 
 %% == EDoc ==
 
-{edoc_opts, [
-  {report_missing_types, true},
-  {source_path, ["src"]},
-  {report_missing_types, true},
-  {todo, true},
-  {packages, false},
-  {subpackages, false}
-]}.
+{
+  edoc_opts,
+  [
+    {report_missing_types, true},
+    {source_path, ["src"]},
+    {report_missing_types, true},
+    {todo, true},
+    {packages, false},
+    {subpackages, false}
+  ]
+}.
 
 %% == Dialyzer ==
 
-{dialyzer, [
-  {warnings, [
-    race_conditions,
-    no_return,
-    unmatched_returns,
-    error_handling
-  ]},
-  {plt_apps, top_level_deps},
-  {plt_extra_apps, []},
-  {plt_location, local},
-  {plt_prefix, "sumo_db_riak"},
-  {base_plt_location, ".plt"},
-  {base_plt_prefix, "sumo_db_riak"}
-]}.
+{
+  dialyzer,
+  [
+    {warnings, [race_conditions, no_return, unmatched_returns, error_handling]},
+    {plt_apps, top_level_deps},
+    {plt_extra_apps, []},
+    {plt_location, local},
+    {plt_prefix, "sumo_db_riak"},
+    {base_plt_location, ".plt"},
+    {base_plt_prefix, "sumo_db_riak"}
+  ]
+}.
 
 %% == Shell ==
 
 {shell, [{apps, [sumo_db_riak]}]}.
+
+%% == Plugins ==
+
+{plugins, [steamroller]}.
```

Comment:
Since this is an inaka project, on principle, we should use `rebar3_format` instead of `steamroller`.
In any case, you can either…
* Fork the project and use `steamroller` on your fork.
* Use `rebar3_format` with a profile for you that uses the `sr_formatter` formatter (that one formats the code with `steamroller`).

Reply:
I wasn't aware of rebar3_format, let  me check it out, Thanks

Reply:
This is totally off-topic, but it's something that I always wonder about…
As a maintainer of multiple open-source tools for the Erlang community (and a blog), I'm always trying to reach as many Erlang devs as possible with the news.
So, 100% **not** as a criticism: If you didn't know about `rebar3_format` but you knew about `steamroller`… How do you get your news about Erlang and its ecosystem? What communication channel am I missing?
I mean… as far as I know, @dtip published `steamroller` in exactly the same places I published `rebar3_format` (sometimes even **I** was the one publishing his work). So… If you don't mind… What did I miss? How can I improve the broadcasting of news about the tools I create/maintain?

Reply:
Might be because I searched for "opinionated code formatter for erlang" 
coming from python as a python dev, I discovered the comfort of black for python
wanted the same for Erlang

Mostly I'm searching for concepts parallel to python

Reply:
Opinionated formatters are the right choice 😉

Reply:
:trollface: